### PR TITLE
Add info on where to find image

### PR DESCRIPTION
**Pull request by zeenix**
NOTE: this is on top of https://github.com/Pelagicore/software-factory/pull/10

### CODE REVIEW

File: docs/chapters/baseplatform/obtaining-a-PELUX-release.rst [hunk 1]
```diff
@@ -14,20 +14,29 @@ Variables:
...
 
-When done fetching the sources, create a build directory and set up bitbake. TEMPLATECONF tells the `oe-init-build-env` script which path to fetch configuration samples from. Note that the example below get the template configuration for the Intel BSP, adapt the path according to your current BSP.
+    mkdir pelux
+    cd pelux
+    repo init -u https://github.com/Pelagicore/pelux-manifests.git -m <manifest> -b <branch>
+    repo sync
 
-.. literalinclude:: snippets/repo-source.sh
-    :language: bash
+When done fetching the sources, create a build directory and set up bitbake. TEMPLATECONF tells the `oe-init-build-env` script which path to fetch configuration samples from.
```
Comment (jonte) [inline docs/chapters/baseplatform/obtaining-a-PELUX-release.rst]: Backticks for TEMPLATECONF

File: docs/chapters/baseplatform/obtaining-a-PELUX-release.rst [hunk 2]
```diff
@@ -14,20 +14,29 @@ Variables:
...
 
-The script will create configs if there are no configs present, a message about created `conf/local.conf` and `conf/bblayers.conf` files is normal.
+.. note:: The example below get the template configuration for the Intel BSP, adapt the path according to your current BSP.
+
+.. code-block:: bash
+
+    TEMPLATECONF=`pwd`/sources/meta-pelux-bsp-intel/conf/ source sources/poky/oe-init-build-env build
 
+The script will create configs if there are no configs present, a message about created `conf/local.conf` and `conf/bblayers.conf` files is normal.
```
Comment (jonte) [inline docs/chapters/baseplatform/obtaining-a-PELUX-release.rst]: Double backticks for paths

File: docs/chapters/baseplatform/obtaining-a-PELUX-release.rst [hunk 3]
```diff
@@ -14,20 +14,29 @@ Variables:
...
 
 Finally, build the desired image. See the variables description above for information on the different images.
 
-.. literalinclude:: snippets/bitbake.sh
-    :language: bash
+.. code-block:: bash
+
+    bitbake <image>
+
+When the build is complete the result will be available in `tmp/deploy/images/<machine>/`. It is possible to generate a number of different image formats, ranging from just the rootfs as a tarball to ready disk-images containing EFI-bootloader, configuration and rootfs and that can be written directly to a storage device. For PELUX, the preffered format for the Intel NUC are .wic images, which are complete disk-images.
```
Comment (jonte) [inline docs/chapters/baseplatform/obtaining-a-PELUX-release.rst]: Double backticks for paths

**Review by jonte — CHANGES_REQUESTED** [review]

File: docs/chapters/baseplatform/obtaining-a-PELUX-release.rst (hunk 1 above)
Comment (jonte) [reply docs/chapters/baseplatform/obtaining-a-PELUX-release.rst]: and oe-init-build..

File: docs/chapters/baseplatform/obtaining-a-PELUX-release.rst (hunk 2 above)
Comment (zeenix) [reply docs/chapters/baseplatform/obtaining-a-PELUX-release.rst]: You reviewed changes from other PR that this PR is based on and that PR is already merged, so I'll add additional patch and when i push the weird situation of having other PR's patches here would also be automatically resolved. Github should have some infra for a PR being based on another one, or does it already?

**Review by breakreturn — APPROVED** [review]

**Review by jonte — APPROVED** [review]

**Pull request merged**